app/client.py

```diff
@@ -0,0 +1,62 @@
+# External imports
+import grpc
+
+# Internal imports
+from app.services.company import CompanyService
+from app.protos.company_pb2 import (
+    CreateCompanyRequest,
+    DeactivateCompanyRequest,
+    ReactivateCompanyRequest,
+)
+
+
+class CompanyClient:
+    def __init__(self, address, token):
+        self.token = token
+        self.channel = self.create_channel(address)
+        self.service = self.create_service()
+
+    def create_channel(self, address):
+        call_credentials = grpc.access_token_call_credentials(self.token)
+        channel_credentials = grpc.ssl_channel_credentials()
+        composite_credentials = grpc.composite_channel_credentials(
+            channel_credentials, call_credentials
+        )
+        return grpc.secure_channel(address, composite_credentials)
```

Comment:
Essa config de SSL ta vazia. Teria q ver com o pessoal de arquitetura ou cloud como q vai ser essa questão de SSL, mas no momento eu diria p/ vc remover isso e deixar só a do token.
```suggestion
        return grpc.secure_channel(address, call_credentials)
```